.github/workflows/update-durations.yml

```diff
@@ -0,0 +1,102 @@
+name: Update Durations Files
+on:
+  # Scheduled trigger on the 15th of every month at 2:47am UTC
+  # schedule:
+  # - cron: '47 2 15 * *'
+  pull_request:
+  #   types:
+  #     - opened
+  #     - reopened
+  #     - synchronize
+  #     - ready_for_review
+  workflow_dispatch:
+
+concurrency:
+  group: unit-tests-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  DURATIONS_BRANCH: bot/durations-update
+
+jobs:
+  update-durations:
+    # if: >-
+    #   ${{
+    #     github.event_name == 'schedule' ||
+    #     github.event_name == 'workflow_dispatch' ||
+    #     (
+    #       github.event_name == 'pull_request' &&
+    #       github.event.pull_request.draft == false &&
+    #       contains(github.event.pull_request.labels.*.name, 'ci:update-durations') &&
+    #       github.event.pull_request.state == 'open'
+    #     )
+    #   }}
+    uses: ./.github/workflows/interface-unit-tests.yml
+    secrets:
+      codecov_token: ${{ secrets.CODECOV_TOKEN }}
+    with:
+      branch: ${{ github.ref }}
+
+      # Run a 'lightened' version of the CI on Pull Requests by default
+      # Unless the label `ci:run-full-test-suite` is attached to the PR.
+      # Always runs the full suite for push events.
+      pipeline_mode: 'update-durations'
+      run_lightened_ci: true
+      skip_ci_test_jobs: 'torch-tests,autograd-tests,all-interfaces-tests,external-libraries-tests,qcut-tests,qchem-tests,gradients-tests,data-tests,device-tests'
```

Comment:
We run lightened CI so that we only run the tests with python 3.10, and the listed tests are skipped (everything except for `tf`, `jax`, and `core`).